ci-operator/jobs/cri-o/cri-o/cri-o-cri-o-release-1.13-postsubmits.yaml

```diff
@@ -0,0 +1,42 @@
+postsubmits:
```

Comment:
You shouldn't need this, remove the promotion config from your job (remove `promotion:`) and then regen and it should go away.

Reply:
I removed `promotion:` from the config yaml and regenerated, but the postsubmits did not go away.

Reply:
delete the new files under ci-operator/jobs/* that got created, then run regenerate

Reply:
got it, thanks! done